operators/index.adoc

```diff
@@ -0,0 +1,41 @@
+[id="operators-overview"]
+= Operators overview
+include::modules/common-attributes.adoc[]
+:context: operators-overview
+
+toc::[]
+
+[id="importance-of-operators"]
+== The importance of Operators
+
+An Operator extends the Kubernetes API. With Operators, you an create applications to monitor the running services in the cluster.
+Operators are designed specifically for your applications. Operators implement and automate the common Day 1 operations such as installation and configuration as well as Day 2 operations such as auto-scaling up and down and backups. All these activities are in a piece of software running inside your cluster.
+To understand more about Operators, see xref:../operators/understanding/olm-what-operators-are.adoc#olm-what-operators-are[What are Operators?]
+
+[id="developer-tasks"]
```

Comment:
Probably good to make this a little more unique, since "developer-tasks" could potentially be a section in some unrelated part of the docs. Something like:

```
[id="operators-overview-developer-tasks"]
```